pytests/test_html.py

```diff
@@ -1,8 +1,40 @@
-
 from subprocess import run
 from helper import get_html_paths
 
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.support.wait import WebDriverWait
+import pytest
+from helper import get_css_absolute_paths, fd_input
+
+def test_html_w3c(driver: WebDriver):
+    files = get_html_paths()
+    for file in files:
+        driver.get("https://validator.w3.org/#validate_by_upload")
+        file_input = WebDriverWait(driver, 10).until(
+            lambda d: d.find_element(By.ID, "uploaded_file")
+        )
+
+        file_input.send_keys(file)
+        file_input.submit()
+
+        WebDriverWait(driver, 10).until(
+            lambda d: d.find_element(By.ID, "results")
+        )
+
```

Comment:
Sprawdzenie czy element ma klasę "success" z użyciem waita wydaję się być bardzo na około. Dodatkowo w przypadku gdy test nie przechodzi czekamy 10s. Lepiej byłoby znaleźć dokładnie element który ma klasę success w przypadku sukcesu i po prostu sprawdzić czy ją ma po zakończeniu testu. Można się posiłkować:
https://stackoverflow.com/questions/39688512/how-to-get-class-name-of-the-webelement-in-python-selenium